src/index.html

```diff
@@ -9,6 +9,60 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+        <nav>
+          <ul>
+            <li>
+              <a href="#"
+              class="is-active"
+              >
```

Comment:
```suggestion
              <a href="#" class="is-active">
```
you can write open-tags in 1 line if there are 1-2 small attributes